ENIGMAsystem/SHELL/Universal_System/mathnc.h

```diff
@@ -38,7 +38,7 @@
 #include "generic_args.h"
 
 #ifndef INCLUDED_FROM_SHELLMAIN
```

Comment:
You can just remove this block, now.

Reply:
Done: 85952fd35fb87d4ecbb50ca58321a7b023f50a1d

Was just making sure.